src/pages/VenuePage/VenuePage.tsx

```diff
@@ -239,6 +239,11 @@ const VenuePage = () => {
       : undefined
   );
 
+  useFirestoreConnect({
+    collection: "venues",
+    storeAs: "venues",
+  });
+
```

Comment:
What is this required here for? It's not immediately obvious from the rest of the changes in this PR.

Reply:
`if (!relevantVisits || !venues) return [];` for one, and venues is always false so its always returning an empty array.

there is several (8 i think) places where we check for `!venues` and in all cases, its false because the key doesnt exist

Reply:
I guess more the point of the question was if this is replacing something that was deleted, where was it removed and why? And if it never existed, how did those features ever work (if they did)?

Reply:
And why add this in `VenuePage` as opposed to closer to where it is needed in `Badges`?

Reply:
Usages (on my local branch) 
![Uploading image.png…]()

Reply:
`useFirestoreConnect("venues")` is the 'canonical' way of doing this, as there is no need for `storeAs` because it uses the name of the collection by default.

Which exists in a few places currently:
![image](https://user-images.githubusercontent.com/753891/101653372-5d9b6280-3a93-11eb-9e47-c54c4de0cb98.png)

So I expect the fix we want is to just add `useFirestoreConnect("venues")` to `src/components/organisms/Badges/Badges.tsx`